_posts/2019-10-29-design-at-hackillinois.md

```diff
@@ -0,0 +1,40 @@
+
+>“We’re not designing products anymore, we’re designing systems that eventually will influence human expectations and enable a change in behavior.”
+
+![HackIllinois 2019 swag items and trophy](/assets/article_images/design-at-hackillinois/2019_swag.jpg)
+
+The newly themed creations begin following the event, getting a full start on the year-long journey of prepping for the hackathon weekend. A small team of student designers works from scratch to collaboratively design every detail of the event that you may interact with: web and mobile, print, swag, social media, and more. The team works long hours to brand each HackIllinois while pushing the limits further each year on what the event is capable of. The small agency feel of the team allows for rapid innovation and purposeful creation. While a design makes an event exciting and visually appealing, the depth of incorporating these styles and assets into the event goes beyond just first look and more into making HackIllinois what it is - an immersive learning and adventurous 48 hours. 
+
+Design is important and all around us. It influences our experiences, our decision making, our mood, and so many other things that you may not expect design to impact. With all these elements at play, design has the ability to transform the basic experience into the events that you talk about for months and years after it has happened. Designers steer the mindsets of those participating to get excited and be invested in open source, hackathons, and collaboration and keep them coming back for the following years. It’s more than posters hanging in the north quad, or the mobile app attendees use, or the amazing swag bags and clothing everyone gets excited about. It’s about creating an environment where the ease of great immersive design enhances students’ and mentors’ abilities to discover and generate incredible projects. Best said from Jan Hellemans, “We’re not designing products anymore, we’re designing systems that eventually will influence human expectations and enable a change in behavior.” Design at HackIllinois pushes the expectations of what open source can do for the world. 
+
+![A group of HackIllinois 2018 staff members take a picture in front of the photobooth in their staff t-shirts and jackets.](/assets/article_images/design-at-hackillinois/2018_photobooth.jpg)
+
+
+As an outsider in the coding community, it’s amazing to see the effect of design on attendees from all over the world. The curious build-up of branding and event designs is filled with thrill when attendees quickly look around to see the details in the Siebel and ECEB Building. When students ask if there are extra pins and stickers for their friends back home or to the intriguing UIUX design of web and mobiles, it definitely puts the cherry on top for all the hard work that our small but mighty team produces. 
```

Comment:
```suggestion
As an outsider to the coding culture, it’s amazing to see the effect of design on attendees from all over the world. The curiosity built up by the branding of the event is exciting. It's thrilling to see attendees interacting with the designs and noticing the details in the Siebel and ECEB Building. When students ask if there are extra pins and stickers for their friends back home or point out intriguing UI/UX features of the website and mobile app, it makes all the hard work worthwhile. 
```

Reply:
"The curious build-up of branding and event designs is filled with thrill when attendees quickly look around to see the details in the Siebel and ECEB Building."

- _"Curious build-up"_ suggests that the process of revealing branding and designs itself is curious or peculiar. 
- _"filled with thrill"_ seemed out of tune with the language used in the rest of the piece. 
- _"quickly look around"_ suggests that they're dismissive of the attention to detail and don't pay much time noticing the work put in.